src/doc/contrib/src/process/index.md

```diff
@@ -8,11 +8,6 @@ process.
 
 Please read the guidelines below before working on an issue or new feature.
 
-**Due to limited review capacity, the Cargo team is not accepting new features
-or major changes at this time. Please consult with the team before opening a
-new PR. Only issues that have been explicitly marked as accepted will be
-reviewed.**
```

Comment:
I feel like this part of the message is buried and indirect in the other sections.  Should we find a way to elevate it?

Reply:
Updated with a link to the `S-accepted` label. What do you think?

Reply:
Thanks!